build/deb/ethereum/deb.install

```diff
@@ -1 +1,2 @@
 build/bin/{{.BinaryName}} usr/bin
+build/deb/ethereum/completions/bash_autocomplete /etc/bash_completion.d/geth
```

Comment:
I think this is not 100% correct. This file is a template and used for all binary packages, not just geth. So we need to conditionalize on `BinaryName == geth` here.

```
{{- if eq .BinaryName "geth" }}
build/deb/ethereum/completions/bash_autocomplete /etc/bash_completion.d/geth
{{end -}}
```